.idea/JavaSMT.iml

```diff
@@ -313,6 +311,15 @@ SPDX-License-Identifier: Apache-2.0
         <SOURCES />
       </library>
     </orderEntry>
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="jar://$USER_HOME$/Downloads/antlr-4.13.1-complete.jar!/" />
```

Comment:
Delete/Change this entry please. It refers to your system. It should look like the other ones with `MODULE_DIR` instead of `USER_HOME`.